### PR TITLE
Add the tengolo cleaver

### PR DESCRIPTION
**Pull request by r4d6**
## About The Pull Request
Add a cleaver that deal extra damage to tengolo, including their berserk variants.

### CODE REVIEW

File: code/game/objects/items/weapons/tools/simple_weapons.dm [hunk 1]
```diff
@@ -338,6 +338,25 @@
...
+	name = "Cleaver Sustine"
+	desc = "A weapon designed by the Hunting Lodge, this massive sword is especially effective against Tengolos, berserk or not."
+	icon = 'icons/obj/weapons-blades.dmi'
+	icon_state = "cleaver"
+	item_state = "cleaver"
+	tool_qualities = list(QUALITY_CUTTING = 30)
+	force = WEAPON_FORCE_BRUTAL
+	armor_penetration = ARMOR_PEN_SHALLOW
+	w_class = ITEM_SIZE_BULKY
+	var/list/effective_faction = list("tengo", "tengolo_berserker") // Which faction the cleaver is effective against.
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/tools/simple_weapons.dm:351]: Build this into base obj please

Comment (r4d6) [reply code/game/objects/items/weapons/tools/simple_weapons.dm:351]: You mean the `effective_faction` ?

Comment (Trilbyspaceclone) [reply code/game/objects/items/weapons/tools/simple_weapons.dm:351]: the damage mulitplyer and the idea of faction effect.
got a few ideas of how to use this on other object/tools

Comment (r4d6) [reply code/game/objects/items/weapons/tools/simple_weapons.dm:351]: Alright.

Comment (r4d6) [reply code/game/objects/items/weapons/tools/simple_weapons.dm]: And done~

**Pull request merged**